src/main/webapp/app/exercises/quiz/shared/questions/drag-and-drop-question/drag-and-drop-question.component.html

```diff
@@ -28,20 +28,21 @@ <h4 class="question-title-display">
         <p [innerHTML]="renderedQuestion.text"></p>
         <span style="color: red" *ngIf="question.invalid" jhiTranslate="artemisApp.quizQuestion.invalidText"></span>
         <div class="hint" *ngIf="question.hint || (question.explanation && showResult)">
-            <ng-template #renderedHint>
-                <div [innerHTML]="renderedQuestion.hint"></div>
-            </ng-template>
-            <span class="label label-info" [ngbPopover]="renderedHint" placement="right" triggers="mouseenter:mouseleave" *ngIf="question.hint">
+            <span class="label label-info" [ngbPopover]="renderedHint" placement="auto" triggers="mouseenter:mouseleave" *ngIf="question.hint">
                 <fa-icon [icon]="['far', 'question-circle']"></fa-icon>
                 <span jhiTranslate="artemisApp.quizQuestion.hint"></span> </span
-            ><br />
-            <ng-template #renderedExplanation>
-                <div [innerHTML]="renderedQuestion.explanation"></div>
+            >
+            <ng-template #renderedHint>
+                <div [innerHTML]="renderedQuestion.hint"></div>
             </ng-template>
-            <span class="label label-primary" [ngbPopover]="renderedExplanation" placement="right" triggers="mouseenter:mouseleave" *ngIf="question.explanation && showResult">
+            <br />
```

Comment:
```suggestion
            <br/>
```
I guess here you missed a whitespace